packages/release-please/src/release-please.ts

```diff
@@ -0,0 +1,99 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Application } from 'probot';
+
+// TODO: fix these imports when release-please exports types from the root
+// See https://github.com/googleapis/release-please/issues/249
+import { ReleaseType, BuildOptions } from 'release-please/build/src/release-pr';
+import { ReleasePRFactory } from 'release-please/build/src/release-pr-factory';
+import { Runner } from './runner';
+
+interface ConfigurationOptions {
+  primaryBranch: string;
+  releaseLabels?: string[];
+  releaseType?: ReleaseType;
+}
+
+const DEFAULT_API_URL = 'https://api.github.com';
+const WELL_KNOWN_CONFIGURATION_FILE = 'release-please.yml';
+const DEFAULT_CONFIGURATION: ConfigurationOptions = {
+  primaryBranch: 'master',
+  releaseLabels: ['autorelease: pending', 'type: process'],
+};
+
+function releaseTypeFromRepoLanguage(language: string | null): ReleaseType {
+  switch (language) {
```

Comment:
I would be tempted to `switch(language.toLowerCase())` rather than the multiple switches.

Reply:
done.